pkg/innerring/processors/netmap/cleanup_table_test.go

```diff
@@ -108,7 +116,7 @@ func TestCleanupTable(t *testing.T) {
 			cnt := 0
 			key := hex.EncodeToString(infos[1].PublicKey())
 
-			require.False(t, c.touch(key, 4)) // one node was updated
+			require.False(t, c.touch(key, 4, mapInfos[key])) // one node was updated
```

Comment:
I suppose this should be `require.True` because `touch()` return value was inverted. Maybe it is related to comment above.

Reply:
Changed along with https://github.com/nspcc-dev/neofs-node/pull/963#discussion_r746622222